all/projectAlcazar/src/app/project.c

```diff
@@ -46,6 +51,23 @@
 #define UART2_TX 2
 #define UART2_RX 3
 
+//HOMEWORK: functia pentru pwm-ul led-urilor de pe placa
+void initBoardPWM(void)
+{
+	//dam enable la portul c de gpio
+	enablePeripheralRCC_AHBENR(ePerif_GPIOC, eEnabled);
+
+	//setam alternate function pentru PC8 + PC9
+	setPinAlternateFunction(GPIOC, LED_PIN_BLUE, ePin_AF2);
```

Comment:
PC8 and PC8 only have ONE alternate function see table 13 in datasheet section 4. So here should be AF0. Only ports A and B have more alternate functions per poin that's why there are 2 tables alternate functions for port A and port B to show which alternate fucntion  is for each value